pkg/management/component.go

```diff
@@ -335,11 +362,26 @@ func (c *component) GetUser(ctx context.Context, realmName, userID string) (api.
 		username = *userKc.Username
 	}
 
+	// Retrieve info from DB user
+	dbUser, err := c.usersDBModule.GetUser(ctx, realmName, *userKc.ID)
+	if err != nil {
```

Comment:
If user is not found, is err nil? We should not return an error if err==sql.ErrNoRows...
(according to the following if statement, I thing that both dbUser and err are nil in such a case)

Reply:
I don't understand the point. If user is not found, the method returns (nil,nil) so if there is an error I return it and few lines after I manage the case if the is no rows (if dbUser == nil).